mmv1/products/clouddeploy/DeliveryPipeline.yaml

```diff
@@ -0,0 +1,178 @@
+    description: "Output only. Time at which the pipeline was created."
+    output: true
+  - !ruby/object:Api::Type::String
+    name: "updateTime"
+    description: "Output only. Time at which the pipeline was updated."
+    output: true
+  - !ruby/object:Api::Type::NestedObject
+    name: "condition"
+    description: "Output only. Information around the state of the Delivery Pipeline."
+    output: true
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: "pipelineReadyCondition"
+        description: "Details around the Pipeline's overall status."
+        output: true
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: "status"
+            description: "True if the Pipeline is in a valid state. Otherwise at least one condition in `PipelineCondition` is in an invalid state. Iterate over those conditions and see which condition(s) has status = false to find out what is wrong with the Pipeline."
+            output: true
+          - !ruby/object:Api::Type::String
+            name: "updatedTime"
+            description: "Last time the condition was updated."
+            output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: "targetsPresentCondition"
+        description: "Details around targets enumerated in the pipeline."
+        output: true
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: "status"
+            description: "True if there aren't any missing Targets."
+            output: true
+          - !ruby/object:Api::Type::Array
+            name: "missingTargets"
+            description: "The list of Target names that do not exist. For example, `projects/{projectId}/locations/{location_name}/targets/{target_name}`."
+            output: true
+            item_type: Api::Type::String
+          - !ruby/object:Api::Type::String
+            name: "updatedTime"
+            description: "Last time the condition was updated."
+            output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: "targetsTypeCondition"
+        description: "Details on the whether the targets enumerated in the pipeline are of the same type."
+        output: true
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: "status"
+            description: "True if the targets are all a comparable type. For example this is true if all targets are GKE clusters. This is false if some targets are Cloud Run targets and others are GKE clusters."
+            output: true
+          - !ruby/object:Api::Type::String
+            name: "errorDetails"
+            description: "Human readable error message."
+            output: true
+  - !ruby/object:Api::Type::String
+    name: "etag"
+    description: "This checksum is computed by the server based on the value of other fields, and may be sent on update and delete requests to ensure the client has an up-to-date value before proceeding."
```

Comment:
If this field will be provided by the API but still an input field, we should add `default_from_api: true`